index.js

```diff
@@ -1,23 +1,14 @@
 #!/usr/bin/env node
```

Comment:
Can you revert your last commit. Because this project's entire point is supporting Node versions, I specifically went with a version of dirname that would be compatible with older versions of Node. The version you changed to only supports Node 20+, while even 18 is still supported/maintained.